### PR TITLE
Add homepage descriptions and entries for Bazarr, Piwigo, PyLoad, and YoutubeDL Material

### PR DESCRIPTION
**Pull request by darshitpp**
**What this PR does / why we need it**:
I noticed that some of the services that I used, did not list themselves on `Homepage`. Adding the same.

### CODE REVIEW

File: roles/piwigo/tasks/main.yml [hunk 1]
```diff
@@ -59,6 +59,11 @@
           traefik.http.routers.piwigo.tls.domains[0].main: "{{ ansible_nas_domain }}"
           traefik.http.routers.piwigo.tls.domains[0].sans: "*.{{ ansible_nas_domain }}"
           traefik.http.services.piwigo.loadbalancer.server.port: "80"
+          homepage.group: Photos
+          homepage.name: Piwigo
+          homepage.icon: piwigo.png
+          homepage.href: "http://{{ hostvars[inventory_hostname]['ansible_default_ipv4']['address'] }}:80"
```
Comment (davestephens) [inline roles/piwigo/tasks/main.yml:65]: Please replace 80 with `{{ piwigo_port}}`

**Review by davestephens — CHANGES_REQUESTED** [review]
Nearly there, just a minor tweak needed :)

File: roles/piwigo/tasks/main.yml (hunk 1 above)
Comment (darshitpp) [reply roles/piwigo/tasks/main.yml]: Fixed

**Comment by davestephens** [comment]
Awesome, thanks!

**Pull request merged**